src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -113,9 +114,31 @@ const ReportParticipantsModalStackNavigator = createModalStackNavigator([
     },
 ]);
 
+// https://reactnavigation.org/docs/stack-navigator/#options
+// We can refactor and move this somewhere else
+const searchNavigationOptions = {
+    transitionSpec: {
+        open: {
+            animation: 'timing',
+            config: {
+                duration: 200,
+                delay: 0,
+            },
+        },
+        close: {
+            animation: 'timing',
+            config: {
+                duration: 300,
+                delay: 0,
+            },
+        },
+    },
+};
```

Comment:
@parasharrajat I noticed some discussion in the slack thread you posted that mentions the smoothness of the in and out animation. Is this what you had in mind? 

I notice the `duration`s are different. Why?

Reply:
Sorry, This change is not what I meant. Also, I don't think this change is acceptable. Changing animation for one of the pages is not good. Page animation should be the same for all pages in the App. 

Calling Gandalf @shawnborton. 

Extraneous, This PR is still in draft and so let the OP prepare it and mark is ready for review. IMO, we should not review a draft PR.

Reply:
> I notice the `durations` are different.
I don't know I just tested with different timings. We should unify this definitely.

Reply:
> Sorry, This change is not what I meant. Also, I don't think this change is acceptable.

I will add a follow-up comment shortly in the PR, explaining why. Please wait!

> Changing animation for one of the pages is not good. Page animation should be the same for all pages in the App.

I agree on this, I was waiting for @Luke9389 to see my changes. I was planning to a discuss on this once he is back. I will address more in a follow-up comment.

> Calling Gandalf @shawnborton.
I was planning too, thanks for this.

> Extraneous, This PR is still in draft and so let the OP prepare it and mark is ready for review. IMO, we should not review a draft PR.

I am not sure about this, but it's okay to check because see if there is any progress or for any clarifications.

Reply:
> Changing animation for one of the pages is not good. Page animation should be the same for all pages in the App.
> 
> > I agree on this, I was waiting for @Luke9389 to see my changes. I was planning to a discuss on this once he is back. I will address more in a follow-up comment.

Ok, so we all agree they should not be set here. Where are animation durations being set currently?

Reply:
Just below the declaration. 
For the search screen, transition animation is part of the navigation options passed it as variable options shown below,
https://github.com/Expensify/App/blob/a649d45cb8046e71ea791d42b82276acb5b8d019/src/libs/Navigation/AppNavigator/ModalStackNavigators.js#L138-L142

used In `createModalStackNavigator` method in the line below
https://github.com/Expensify/App/blob/a649d45cb8046e71ea791d42b82276acb5b8d019/src/libs/Navigation/AppNavigator/ModalStackNavigators.js#L57

This is how animation is set.

Reply:
> Ok, so we all agree they should not be set here. 

Yes, not like how I did here.

Should move these [lines](https://github.com/Expensify/App/blob/a649d45cb8046e71ea791d42b82276acb5b8d019/src/libs/Navigation/AppNavigator/ModalStackNavigators.js#L117-L136) to a separate file, 
1. Refactor it as a function to make it reusable for other screens providing customize options based on each screen & use it for those screens alone. 

2. Just create a CONST variable with a common name use it here or anywhere else if there is no screen-specific customization need case.

Reply:
Rather than just suggesting these changes in a comment, please do them so we can see the code changes.